build/constants.yaml

```diff
@@ -217,6 +221,10 @@
     node-app-insights-sdk-version: 1.8.7
     pm2-version: 4.5.6
     npm-version: 6.14.15
+    runtime-versions: [4.4, 4.5, 4.8, 6, 6.2, 6.6, 6.9, 6.10,
+                      6.11, 8, 8.0, 8.1, 8.2, 8.8, 8.9, 8.11,
+                      8.12, 9.4, 10, 10.1, 10.10, 10.12, 10.14,
+                      12, 14, 16]
```

Comment:
It might be good to keep all runtime versions consistent by having `major.minor` or only `major` in here.

Reply:
When runnning `oryx dockerfile` command, does the specified platform version need to match with one of listed above?

Reply:
> It might be good to keep all runtime versions consistent by having major.minor or only major in here.

It does look a little odd, but what's listed for each `runtime-versions` property are the runtime image tags that we have supported for each language, and in some cases (specifically node), there is a single value for the runtime image tag (_i.e._, `node:10` is the runtime image)

> When runnning `oryx dockerfile` command, does the specified platform version need to match with one of listed above?

No -- if the user provides a runtime version, we will use that in the Dockerfile no matter what. This is to allow us to publish new runtime versions and the user not having to update the version of their CLI image to be able to use the new versions; the Dockerfile will be populated with whatever version they provide and we'll respect it (regardless of if it's in the CLI image's supported runtime version list or not).